disboardreminder/disboardreminder.py

```diff
@@ -427,46 +424,26 @@ def validate_cache(self, message: discord.Message) -> Optional[discord.TextChann
             return
         return guild.get_channel(bump_chan_id)
 
-    def validate_success(self, message: discord.Message) -> Optional[discord.Embed]:
-        if not message.embeds:
-            return
-        embed = message.embeds[0]
-        if ":thumbsup:" in embed.description:
-            return embed
-        if message.webhook_id and "Bump done!" in embed.description:
-            # slash command responses to the bump command don't have the thumbsup emoji in them
-            # for some reason
-            # this solution is a temporary fix, since it isn't language agnostic, but atm I can't
-            # a different telling sign that only appears on bump command responses
-            return embed
+    def validate_success(self, message: discord.Message) -> bool:
+        return all(
```

Comment:
I believe this is missing a pair of parentheses, since [`all`](https://docs.python.org/3/library/functions.html#all) takes an [`iterable`](https://docs.python.org/3/glossary.html#term-iterable), not varargs.

Just using the `and` keyword might be better so you're not eagerly evaluating every condition before checking, but that'd be a very minor optimization.

Reply:
ahh yes I was using the and keyword previously but it looked ugly codewise so I made a last minute change using the all function. The paranthesis have been added.